django/thunderstore/repository/models/cache.py

```diff
@@ -22,16 +22,26 @@ class APIV1PackageCache(S3FileMixin):
 
     @classmethod
     def get_latest_for_community(
-        cls, community: Optional[Community]
+        cls,
+        community: Optional[Community] = None,
```

Comment:
I'd rather this method _only_ accepts `community__identifier` instead of conditionally swapping, the performance-loss is negligible and that's what we'd want to do in the future anyway.